### ISSUE TITLE
[CI] Support GitHub PR commands other than `/fix` such as `/test`

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#5570]
Our `/fix` series of commands are proving useful. I think that we should support other commands such as:

- `/test` (rather than, as we currently do, ask contributors to run a `fix:all`)
- `/help` or `/list` to list the commands available and/or link to an appropriate Contributing page in our site docs.

I also have others in mind that will be helpful for managing the refcache (I'll fill in details about that later).

**Comment by svrnm** [comment]
A `/help` command would be especially helpful, also to guide end users.

**Comment by svrnm** [comment]
I still think this would be helpful:-)

**Comment by chalin** [comment]
Yeah. A rework & refactoring of our `.github` workflows etc is in order _before_ (or as an initial part of) this IMHO. The idea it to uniformly enforce our trusted / untrusted split of work.
